### DISCUSSION TITLE
Windows 客户端版本号错误

### DISCUSSION THREAD

**Discussion by smallg0at** [xiaoyaocz/flutter_dmzj#122] [General]
2.0.2 的  release 中所有 Windows 客户端都认为自己是 2.0.1，~~建议确认一下是否传错了~~ 确认只是版本号问题，功能上是 2.0.2。

另外，使用触摸键盘选中任意文本框后取消选中，在该文本框离开屏幕后，在应用内的每一次点击都会拉起触摸键盘。

**Comment by xiaoyaocz** [comment]
a33462231567ba5d3bb96d1134bc68b7d02f3233

**Reply by xiaoyaocz** [reply]
键盘拉起似乎是Flutter的BUG https://github.com/flutter/flutter/issues/97269

**Comment by xiaoyaocz** [comment]
TODO:
- [x] 修复Windows版本号问题 https://github.com/xiaoyaocz/flutter_dmzj/commit/a33462231567ba5d3bb96d1134bc68b7d02f3233
- [ ]  Windows触摸键盘拉起问题 https://github.com/flutter/flutter/issues/97269
